python_scripts/trees.py

```diff
@@ -81,19 +81,19 @@
 # We can first check the feature distributions by looking at the diagonal plots
 # of the pairplot. We can build the following intuitions:
 #
-# * The Adelie specie is separable from the Gentoo and Chinstrap species using
+# * The Adelie species is separable from the Gentoo and Chinstrap species using
 #   the culmen length;
-# * The Gentoo specie is separable from the Adelie and Chinstrap species using
+# * The Gentoo species is separable from the Adelie and Chinstrap species using
 #   the culmen depth.
 #
-# ## How decision tree are built?
+# ## How are decision tree built?
 #
 # In a previous notebook, we learnt that a linear classifier will define a
 # linear separation to split classes using a linear combination of the input
 # features. In our 2-dimensional space, it means that a linear classifier will
-# defined some oblique lines that best separate our classes. We define a
-# function below that given a set of data point and a classifier will plot the
-# decision boundaries learnt by the classifier.
+# define some oblique lines that best separate our classes. We define a
```

Comment:
We only demonstrated binary classification in linear model. Maybe we could give more details about how these oblique lines work/look like for >2 classes. e.g., is there one line for each class?

Also maybe explain how these oblique separating lines relate to the decision boundaries..